lib/components/molecules/Pagination/PaginationList.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Button from '../../atoms/Button';
+
+const LEFT_PAGE = 'LEFT';
+const RIGHT_PAGE = 'RIGHT';
+
+const PaginationList = ({
+  pageNumbers,
+  executePagination,
+  activePage,
+  currentPageAria,
+  previousButtonAria,
+  disabledFirstNdPrevious,
+  handleBackwardClick,
+  previousLabel,
+  nextButtonAria,
+  disabledNextAndLast,
+  handleForwardClick,
+  nextLabel,
+}) => {
+  return (
+    pageNumbers &&
+    pageNumbers.map(page => {
+      if (page === LEFT_PAGE)
+        return (
+          <li className="previous">
+            <Button
+              aria-label={previousButtonAria}
+              disabled={disabledFirstNdPrevious}
```

Comment:
disabledFirstNdPrevious --> noFirstPreviousCTA

Reply:
Changed the name in both places